modules/andes-core/client/src/main/java/org/wso2/andes/client/AMQSession.java

```diff
@@ -1855,7 +1853,7 @@ public void recover() throws JMSException
             // Set inRecovery to false before you start message flow again again.            
             _inRecovery = false; 
             
-            if (!isSuspended)
+            if (_suspended)
```

Comment:
In the previous code we are checking the negation of suspend. Why have we changed that?